dotcms-integration/src/test/java/com/dotcms/MainSuite2b.java

```diff
@@ -385,6 +387,8 @@
         JobQueueManagerAPITest.class,
         ConfigUtilsTest.class,
         SimpleInjectionIT.class,
+        SimpleDataProviderWeldRunnerInjectionIT.class,
```

Comment:
why did the order of test executions changed?

Reply:
no particular reason

Reply:
we have to make sure it doesn't trigger flakey tests in those old tests that are executed later